hazelcast/src/main/java/com/hazelcast/spi/OperationAccessor.java

```diff
@@ -43,6 +44,11 @@ public static boolean isMigrationOperation(Operation op) {
                 && op.getClass().getClassLoader() == THIS_CLASS_LOADER;
     }
 
+    public static boolean isWanReplicationOperation(Operation op) {
+        return op instanceof WanReplicationOperation
+                && op.getClass().getClassLoader() == THIS_CLASS_LOADER;
```

Comment:
Why do you add this to the OperationAccessor? And why does it matter which classloaded loaded it?